### PR TITLE
add Tab system from bootstrap in SearchUserTable Userdata

### LINKED ISSUE #1668: 🚀 [Feature] Admin make User Details tabbed
## 🚀 Feature

Have the user details tabbed.

The user details in the admin gets to long, have a horizontal divider like tabs to make it not a nightmare to scroll.

Consider to have the creation outside the tabs/as first tab since thats the main freature for that admin list

### PR DESCRIPTION
**Pull request by ogerly**
### Issues
- fixes #1668

### CODE REVIEW

File: admin/src/components/Tables/SearchUserTable.vue [hunk 1]
```diff
@@ -49,30 +49,41 @@
...
-                ? $d(new Date(row.item.emailConfirmationSend), 'long')
-                : ''
-            "
-          />
-          <creation-transaction-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
-          <transaction-link-list :userId="row.item.userId" />
-
-          <deleted-user-formular :item="row.item" @updateDeletedAt="updateDeletedAt" />
+          <b-tabs content-class="mt-3">
+            <b-tab title="Creations" active :disabled="row.item.deletedAt">
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:53]: what happens if you open this and the user is deleted?

**Comment by ogerly** [comment]
if an account is deleted then these tabs are set to disabled

File: admin/src/components/Tables/SearchUserTable.vue [hunk 2]
```diff
@@ -49,30 +49,41 @@
...
-                ? $d(new Date(row.item.emailConfirmationSend), 'long')
-                : ''
-            "
-          />
-          <creation-transaction-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
-          <transaction-link-list :userId="row.item.userId" />
-
-          <deleted-user-formular :item="row.item" @updateDeletedAt="updateDeletedAt" />
+          <b-tabs content-class="mt-3">
+            <b-tab title="Creations" active :disabled="row.item.deletedAt !== null">
```
Comment (Mogge) [inline admin/src/components/Tables/SearchUserTable.vue:53]: Please localize the title

File: admin/src/components/Tables/SearchUserTable.vue [hunk 3]
```diff
@@ -49,30 +49,41 @@
...
+                    ? $d(new Date(row.item.emailConfirmationSend), 'long')
+                    : ''
+                "
+              />
+            </b-tab>
+            <b-tab title="ListCreations" :disabled="row.item.deletedAt !== null">
+              <creation-transaction-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
+            </b-tab>
+            <b-tab title="ListLinks" :disabled="row.item.deletedAt !== null">
+              <transaction-link-list :userId="row.item.userId" />
```
Comment (Mogge) [inline admin/src/components/Tables/SearchUserTable.vue:81]: Use the v-if here as well. If not, the Links are queried for a deleted user

**Review by Mogge — CHANGES_REQUESTED** [review]

**Comment by ogerly** [comment]
@Mogge  locales all tabs
FireShot Capture 1113 - Gradido Admin Interface - localhost

**Review by Mogge — APPROVED** [review]

File: admin/src/components/Tables/SearchUserTable.vue [hunk 4]
```diff
@@ -49,30 +49,44 @@
...
-            "
-          />
-          <creation-transaction-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
-          <transaction-link-list :userId="row.item.userId" />
-
-          <deleted-user-formular :item="row.item" @updateDeletedAt="updateDeletedAt" />
+          <b-tabs content-class="mt-3">
+            <b-tab :title="$t('creation')" active :disabled="row.item.deletedAt !== null">
+              <creation-formular
+                v-if="!row.item.deletedAt"
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:55]: do we still need this?

File: admin/src/components/Tables/SearchUserTable.vue [hunk 5]
```diff
@@ -49,30 +49,44 @@
...
+                type="singleCreation"
+                pagetype="singleCreation"
+                :creation="row.item.creation"
+                :item="row.item"
+                :creationUserData="creationUserData"
+                @update-user-data="updateUserData"
+              />
+              <div v-else>{{ $t('userIsDeleted') }}</div>
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:63]: do we still need this?

File: admin/src/components/Tables/SearchUserTable.vue [hunk 6]
```diff
@@ -49,30 +49,44 @@
...
+            </b-tab>
+            <b-tab :title="$t('e_mail')" :disabled="row.item.deletedAt !== null">
+              <confirm-register-mail-formular
+                v-if="!row.item.deletedAt"
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:67]: do we still need this?

File: admin/src/components/Tables/SearchUserTable.vue [hunk 7]
```diff
@@ -49,30 +49,44 @@
...
+                :email="row.item.email"
+                :dateLastSend="
+                  row.item.emailConfirmationSend
+                    ? $d(new Date(row.item.emailConfirmationSend), 'long')
+                    : ''
+                "
+              />
+            </b-tab>
+            <b-tab :title="$t('creationList')" :disabled="row.item.deletedAt !== null">
+              <creation-transaction-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:78]: do we still need this `if`?

File: admin/src/components/Tables/SearchUserTable.vue [hunk 8]
```diff
@@ -49,30 +49,44 @@
...
+            </b-tab>
+            <b-tab
+              :title="$t('transactionlink.form_header')"
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:81]: this locale seems to be off. Use a self-describing string please.

File: admin/src/components/Tables/SearchUserTable.vue [hunk 9]
```diff
@@ -49,30 +49,44 @@
...
+              :disabled="row.item.deletedAt !== null"
+            >
+              <transaction-link-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
```
Comment (ulfgebhardt) [inline admin/src/components/Tables/SearchUserTable.vue:84]: do we still need this `if`?

File: admin/src/components/Tables/SearchUserTable.vue (hunk 7 above)
Comment (Mogge) [reply admin/src/components/Tables/SearchUserTable.vue:78]: Yes

File: admin/src/components/Tables/SearchUserTable.vue (hunk 9 above)
Comment (Mogge) [reply admin/src/components/Tables/SearchUserTable.vue:84]: Yes

File: admin/src/components/Tables/SearchUserTable.vue (hunk 6 above)
Comment (Mogge) [reply admin/src/components/Tables/SearchUserTable.vue:67]: Yes

File: admin/src/components/Tables/SearchUserTable.vue (hunk 4 above)
Comment (Mogge) [reply admin/src/components/Tables/SearchUserTable.vue:55]: Yes

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**